src/app/blocks/rename-fields-block/rename-fields-block.component.ts

```diff
@@ -41,7 +43,15 @@ export class RenameFieldsBlockComponent  extends BaseBlockComponent {
   mapItem(item, fieldMapping) {
     const output = {};
     (fieldMapping || []).forEach(({ source, target }) => {
-      output[target] = item[source];
+      //set default mapping direction to sourceToTarget
+      let from = source;
+      let to = target;
+      if (this.direction == 'targetToSource') {
```

Comment:
This seems to be a new feature, not just a bug fix. This could be explained better.